app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -436,6 +438,13 @@ class MainActivity : AppCompatActivity(), MainViewController, RouteCallback,
         val cacheSearches = PreferenceManager.getDefaultSharedPreferences(this)
                 .getBoolean(AndroidAppSettings.KEY_CACHE_SEARCH_HISTORY, true)
         searchView?.setCacheSearchResults(cacheSearches)
+        val closeButton = searchView?.findViewById(R.id.search_close_btn) as ImageView
+        closeButton.setOnClickListener {
```

Comment:
Nice job.

Do you think it would be possible to either: 1. Hide the close button when viewing the search results list or 2. Override the behavior to clear the current set of results? Thoughts about which would be better?

It just feels kind of broken right now that you can see the close button but it doesn't do anything when clicked in this mode.